src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl

```diff
@@ -763,6 +951,9 @@
           "email": {
             "type": "string"
           },
+          "authToken": {
+            "type": "string"
+          },
```

Comment:
what is this authToken used for? please remove it

Reply:
must be a copy paste leftover, removed here 9ea6c504eff4d02cf549703d2f86f3040dea8f0a